final_prep/annotations.rb

```diff
@@ -4,40 +4,70 @@
 
 # Build a Bear
 
+# creates a function build_a_bear that takes in five arugments: name, age, fur, clothes, special_power
 def build_a_bear(name, age, fur, clothes, special_power)
+  # initialize greeting variable to a string that interpolates name
   greeting = "Hey partner! My name is #{name} - will you be my friend?!"
+  # initialize demographics variable to an array of name and age
   demographics = [name, age]
+  # initialize power_saying variable to a string that interpolates special_power
   power_saying = "Did you know that I can #{special_power}?"
+  # initialize built_bear variable to a hash
   built_bear = {
+    # basic_info (key) is set to the demographics array
     'basic_info' => demographics,
+    # clothes key is set to the clothes arugment
```

Comment:
Nice job with your line by line annotations! Way to point out whether the value comes from an argument, or a variable declared above.